connectors/spark/src/main/scala/io/unitycatalog/spark/auth/AwsVendedTokenProvider.java

```diff
@@ -0,0 +1,58 @@
+package io.unitycatalog.spark.auth;
+
+import io.unitycatalog.spark.UCHadoopConf;
+import java.net.URI;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.shaded.com.google.common.base.Preconditions;
```

Comment:
Since we will need to deploy this `unitycatalog-spark` jar and users need to add it as the dependency, then I will suggest to use the unity catalog relocated guava for the OSS-unitycatalog release. 

But in this PR, I don't want to block it. So I temporarily use the hadoop's relocated guava.  FYI @tdas